src/tools/types.ts

```diff
@@ -170,6 +172,7 @@ export const MatchingImages: {
 export interface serviceItem {
```

Comment:
@Meierschlumpf can we remove this now?

Reply:
Yes this and the Config interface where it is used can be removed